system/jlib/jstats.cpp

```diff
@@ -112,6 +112,7 @@ static const StatisticScopeType scoreOrder[] = {
     SSTworkflow,
     SSTchildgraph,
     SSTfile,
+    SSTchannel,  // MORE - not sure what this means!
```

Comment:
Do you now?

Reply:
I don't know what scoreOrder is for, no...

Reply:
nor do I really!  I think it is something to do with making sure that stats from different sources come out in a consistent order.